src/notify/subscription/db.go

```diff
@@ -42,13 +37,12 @@ func get(id string) (Subscription, error) {
 	defer rows.Close()
 
 	if rows.Next() {
-		err := scan(rows, &subscription)
-		if err != nil {
+		if err := scan(rows, &subscription); err != nil {
 			return subscription, err
 		}
 	}
 
-	return subscription, nil
+	return subscription, errors.New("No subscription found")
```

Comment:
I guess it is the Rails difference between `find_by` and `find_by!` 

Should the case where a call to `get` by Primary Key lead to an error? I prefer *yes* 
:)